urnai/sc2/environments/stablebaselines3/custom_env.py

```diff
@@ -0,0 +1,51 @@
+import gymnasium as gym
+from gymnasium import spaces
+
+from urnai.actions.action_space_base import ActionSpaceBase
+from urnai.environments.environment_base import EnvironmentBase
+from urnai.rewards.reward_base import RewardBase
+from urnai.states.state_base import StateBase
+
+
+class CustomEnv(gym.Env):
+    """Custom Environment that follows gym interface."""
+
+    metadata = {"render_modes": ["human"], "render_fps": 30}
+
+    def __init__(self, env: EnvironmentBase, state: StateBase, 
+                 urnai_action_space: ActionSpaceBase, reward: RewardBase, 
+                 observation_space: spaces.Space, action_space: spaces.Space):
+        super().__init__()
+
+        self._env = env
+        self._state = state
+        self._action_space = urnai_action_space
+        self._reward = reward
+        self._obs = None
+        # SB3 spaces
```

Comment:
What is `SB3`?

Reply:
Should be Stable Baselines3, will change in the next commit

Reply:
Actually, these spaces are from `gymnasium` :D
What I wanted in the comment was to say that these variables are used internally by the library